static/js/map.js

```diff
@@ -0,0 +1,16 @@
+function initialize() {
+  var myLatlng = new google.maps.LatLng(-1.9446696,30.089721);
+  var mapOptions = {
+    zoom: 14,
+    center: myLatlng
+  }
+  var map = new google.maps.Map(document.getElementById('klab-map'), mapOptions);
+  var marker = new google.maps.Marker({
+      position: myLatlng,
+      map: map,
+      title: 'kLab',
+      icon: image
```

Comment:
change the varible name image to something like `kLabImageMapLogo` instead of just image